File: Beecrowd/1145 Sequência Lógica 2/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        int x, y, aux;

        x = sc.nextInt();
        y = sc.nextInt();
        aux = x;
        for (int i = 1; i <= y; i++) {
            if (i < x)
                System.out.print(i + " ");
            else {
                System.out.println(i);
                x += aux;
            }
        }

        sc.close();
    }
}

File: Beecrowd/1145 Sequência Lógica 2/main.cpp
#include <iostream>
 
using namespace std;

int main(){
  int X, Y, i, aux;

  cin >> X >> Y;

  aux = X;

  for(i = 1; i <= Y; i++){
    if(i < X)
      cout << i << " ";
    else{
      cout << i << endl;
      X += aux;
    }
  }
  return 0;
}